src/styles/globals.css

```diff
@@ -3,51 +3,94 @@
 
 @font-face {
-    font-family: 'neue-haas';
-    src: url('/fonts/NeueHaasDisplay/NeueHaasDisplayThin.ttf') format('truetype');
-    font-weight: 100;
+  font-family: "neue-haas";
+  src: url("/fonts/NeueHaasDisplay/NeueHaasDisplayThin.ttf") format("truetype");
+  font-weight: 100;
 }
 
 @font-face {
-    font-family: 'neue-haas';
-    src: url('/fonts/NeueHaasDisplay/NeueHaasDisplayLight.ttf') format('truetype');
-    font-weight: 300;
+  font-family: "neue-haas";
+  src: url("/fonts/NeueHaasDisplay/NeueHaasDisplayLight.ttf") format("truetype");
+  font-weight: 300;
 }
 
 @font-face {
-    font-family: 'neue-haas';
-    src: url('/fonts/NeueHaasDisplay/NeueHaasDisplayRoman.ttf') format('truetype');
-    font-weight: 400;
+  font-family: "neue-haas";
+  src: url("/fonts/NeueHaasDisplay/NeueHaasDisplayRoman.ttf") format("truetype");
+  font-weight: 400;
 }
 
 @font-face {
-    font-family: 'neue-haas';
-    src: url('/fonts/NeueHaasDisplay/NeueHaasDisplayBlack.ttf') format('truetype');
-    font-weight: 900;
+  font-family: "neue-haas";
+  src: url("/fonts/NeueHaasDisplay/NeueHaasDisplayBlack.ttf") format("truetype");
+  font-weight: 900;
 }
 
 @font-face {
-    font-family: 'neue-haas';
-    src: url('/fonts/NeueHaasDisplay/NeueHaasDisplayMedium.ttf') format('truetype');
-    font-weight: 500;
+  font-family: "neue-haas";
+  src: url("/fonts/NeueHaasDisplay/NeueHaasDisplayMedium.ttf")
+    format("truetype");
+  font-weight: 500;
 }
 
 @font-face {
-    font-family: 'neue-haas';
-    src: url('/fonts/NeueHaasDisplay/NeueHaasDisplayBold.ttf') format('truetype');
-    font-weight: 700;
+  font-family: "neue-haas";
+  src: url("/fonts/NeueHaasDisplay/NeueHaasDisplayBold.ttf") format("truetype");
+  font-weight: 700;
 }
 
 @font-face {
-    font-family: 'neue-haas';
-    src: url('/fonts/NeueHaasDisplay/NeueHaasDisplayLightItalic.ttf') format('truetype');
-    font-style: italic;
+  font-family: "neue-haas";
+  src: url("/fonts/NeueHaasDisplay/NeueHaasDisplayLightItalic.ttf")
+    format("truetype");
+  font-style: italic;
 }
 
 @layer components {
-    #__next {
-        display: flex;
-        flex-direction: column;
-        min-height: 100vh;
-    }
+  #__next {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
+  }
+}
+
+/* For Webkit-based browsers (Chrome, Safari and Opera) */
+.scrollbar-hide::-webkit-scrollbar {
+  display: none;
+}
+
+/* For IE, Edge and Firefox */
+.scrollbar-hide {
+  -ms-overflow-style: none;  /* IE and Edge */
+  scrollbar-width: none;  /* Firefox */
+}
+
+.selectedStyle {
+  cursor: pointer;
+  fill: #3d84e8;
+  transition: all 700ms ease-in-out;
+  z-index: 1000;
+  filter: drop-shadow(0 0 7px #ee2a7b);
+  animation: blinkingText 2s infinite;
+}
+
+@keyframes blinkingText {
+  0% {
+    fill: #ee2a7b;
+    r: 23.57px;
+  }
+  25% {
+    fill: #ee2a7b;
+  }
+  50% {
+    fill: #ee2a7b;
+    r: 26.57px;
+  }
+  75% {
+    fill: #ee2a7b;
+  }
+  100% {
+    fill: #ee2a7b;
+    r: 23.57px;
+  }
```

Comment:
Since the fill is the same at all points here, can't it just be removed and added to the element instead? Also, this can be put into the tailwind config instead (search for keyframes on [this page](https://tailwindcss.com/docs/animation)), but I'm not 100% convinced that it is preferable.